install_config/topics/full_rhv_inventory.adoc

```diff
@@ -4,21 +4,18 @@ Module included in the following assemblies:
 install_config/configuring_rhv.adoc
 ////
 
-= Configuring {product-title} for Red Hat Virtualization 
+= Configuring {product-title} for Red Hat Virtualization
 
-You configure {product-title} for Red Hat Virtualization by modifying the
-Ansible inventory file before you install the cluster.
+You can configure {product-title} for Red Hat Virtualization by modifying the Ansible inventory file before you install the cluster.
 
```

Comment:
I'm confused about your structure. At this point in the assembly, you've already installed OpenShift. Why do we have another hosts file and set of installation commands here?

There are a few options that might be more appropriate, but I'm not sure which is best:
- Create a separate assembly, since this is a separate workflow
- Merge the hosts files
- Turn this into a reference instead of a procedure and explain why it's different than the other hosts file

Reply:
It is a reference module. How do you people distinguish a reference module from any other module?

Reply:
By not including steps in it or titling it like a procedure - only procedure modules have steps and gerunds in the title.